docs/usage.md

```diff
@@ -436,15 +463,18 @@ Triage crashes found by AFL++
 instances, `casr-afl` generates crash reports depending on target binary. For
 binary with ASAN `casr-san` is used, otherwise `casr-gdb`. On the next step report
 deduplication is done by `casr-cluster`. Finally, reports are traiged into
-clusters. Crash reports contain many useful information: severity (like [exploitable](https://github.com/jfoote/exploitable)), OS and package versions, command line, stack trace, register values,
-disassembly, and even source code fragment where crash appeared.
+clusters. Crash reports contain many useful information: severity
+(like [exploitable](https://github.com/jfoote/exploitable)), OS and package
+versions, command line, stack trace, register values, disassembly, and even
+source code fragment where crash appeared. `casr-afl` also provides integration with AFL-based
+fuzzer [Sharpfuzz](https://github.com/Metalnem/sharpfuzz).
 
 **NOTE:** `casr-gdb` and `casr-san` should be in PATH to make `casr-afl` work.
 
-Example (Ubuntu 20.04+):
+AFL++ Example (Ubuntu 20.04+):
 
-    $ cp casr/tests/casr_tests/bin/load_afl /tmp/load_afl
-    $ cp casr/tests/casr_tests/bin/load_sydr /tmp/load_sydr
+    $ cp -r casr/tests/casr_tests/bin/load_afl /tmp/load_afl
```

Comment:
`load_afl` is binary, `-r` option is unnecessary

Reply:
done